package.json

```diff
@@ -109,6 +109,7 @@
     "@commitlint/cli": "^8.0.0",
     "@commitlint/config-conventional": "^8.0.0",
     "@commitlint/travis-cli": "^8.0.0",
+    "@elastic/elasticsearch": "elastic/elasticsearch-js#update-events",
```

Comment:
If this is merged into master, it will break ones the branch is deleted. But for now it's the only way to test our compatibility with the yet unreleased `prepare-request` event which will ship in 7.4.

Maybe we should wait to merge this PR until that PR branch has been merged?